avro-fastserde/src/main/java/com/linkedin/avro/fastserde/PrimitiveFloatList.java

```diff
@@ -159,7 +159,7 @@ public Float get(int i) {
    *
    * TODO: verify if it's enough to simply store the data as primitives in order to benefit from the GC optimization.
    */
-  private boolean addPrimitive(float o) {
+  public boolean addPrimitive(float o) {
```

Comment:
Why did we need this change? It doesn't seem related to this commit, right?

Reply:
This is required by LinkedIn internal application.